storage/integration_test.go

```diff
@@ -383,6 +383,17 @@ func TestIntegration_BucketUpdate(t *testing.T) {
 	if !testutil.Equal(attrs.Lifecycle, wantLifecycle) {
 		t.Fatalf("got %v, want %v", attrs.Lifecycle, wantLifecycle)
 	}
+	// Check that StorageClass has "STANDARD" for unset field.
+	wantStorageClass := "STANDARD"
+	if !testutil.Equal(attrs.StorageClass, wantStorageClass) {
+		t.Fatalf("got %v, want %v", attrs.StorageClass, wantStorageClass)
+	}
```

Comment:
Not sure I'm getting what's going on here. You're checking if `attrs` has `StorageClass` equal to `want`, but how it'll get into `attrs`?

Reply:
If we're not passing any value in StorageClass, the StorageClass will be "STANDARD" by default in attrs metadata. We're just checking it before sending new value "NEARLINE".

Reply:
Maybe this check is redundant but this way we could be 100% confident in a result. But I'm not insist on it, if you think that will be better not to write this check

Reply:
Ha, okay, default value, I got it